solr/core/src/java/org/apache/solr/cli/RunExampleTool.java

```diff
@@ -326,98 +326,98 @@ protected void runExample(CommandLine cli, String exampleName) throws Exception
             "exampledocs directory not found, skipping indexing step for the techproducts example");
       }
     } else if ("films".equals(exampleName) && !alreadyExists) {
-      SolrClient solrClient = new Http2SolrClient.Builder(solrUrl).build();
+      try (SolrClient solrClient = new Http2SolrClient.Builder(solrUrl).build()) {
+        echo("Adding dense vector field type to films schema \"_default\"");
+        try {
+          SolrCLI.postJsonToSolr(
```

Comment:
Look at all the string handling for JSON!  I wondered about maybe using the FieldTypeDefinition.java, but after looking at how it's used, it seems to add another level of complexity/obfuscation over what is happening.   I hope our future V2 api's make doing thinks like adding a field type simpler and cleaner ;-).